python/ray/data/_internal/execution/interfaces.py

```diff
@@ -179,6 +236,13 @@ def __str__(self) -> str:
         out_str += f"{self.__class__.__name__}[{self._name}]"
         return out_str
 
+    def progress_str(self) -> str:
```

Comment:
sorry I don't see caller of `progress_str` in this PR. how this is called?

Reply:
It's called by the new streaming executor to report progress on operators. I'll also add it to bulk.